README.Rmd

```diff
@@ -0,0 +1,155 @@
+```
+
+To load the package, use:
+
+```{r}
+#| label: load-package
+library(srvyrexploR)
+```
+
+## About the data
+
+This package includes data from three surveys including the American National Election Studies (ANES), the National Crime Victimization Survey (NCVS), and the Residential Energy Consumption Survey (RECS). 
+
+### ANES
+
+The ANES data is based on the publicly available 2020 ANES data with additional derived variables and is subset to people who completed both pre and post-election interviews. The ANES Times Series Studies collect data on political polling in the United States and has been conducted since 1948.For more information about the 2020 study, see the [American National Election Studies website](https://electionstudies.org/data-center/2020-time-series-study/). At that site, you can learn more about the study, see codebooks and methodology reports, and download the data (after registering). We received permission to distribute this data for the purpose of the book. Once the package is loaded, you can use the data immediately as follows:
+
+```{r}
+#| label: show-anes
+
+head(anes_2020)
+```
+
+See `?anes_2020` for more information about the data.
+
+Also, included in the package is a Stata version of the ANES data with a subset of the columns and is subset to people who completed both pre and post-election interviews. To load this dataset, we recommend using the **haven** package as follows:
+
+```{r}
+#| label: anes-stata
+
+anes_stata <- haven::read_dta(system.file("extdata", "anes_2020_stata_example.dta", package="srvyrexploR"))
+```
+
+### NCVS
+
+The NCVS data is based off of publicly available data for the 2021 NCVS. The NCVS is a survey conducted by the Bureau of Justice Statistics and asks people age 12 and over about their crime victimizations. The study has been conducted continuously since 1992. This package includes three datasets - one for household-level data (`ncvs_2021_household`), one for person-level data (`ncvs_2021_person`), and one for incident-level data (`ncvs_2021_incident`) where each includes a subset of the columns of the full data available from 2021 at [ICPSR](https://www.icpsr.umich.edu/web/NACJD/studies/38429). This data is reproduced here with permission from ICPSR. 
+
+```{r}
+#| label: ncvs-show
+
+head(ncvs_2021_household)
+head(ncvs_2021_person)
+head(ncvs_2021_incident)
+```
+
+### RECS
+
+Three files are included associated with RECS - a dataset with the 2015 data with some derived variables created for the book (`recs_2015`), the 2020 data with some derived variables created for the book (`recs_2020`), and the 2020 data with the original variables. RECS is a survey about energy consumption and expenditure among residential households in the United States and has been conducted since 1979 by the Energy Information Administration. More information about the original data is available at the [RECS website](https://www.eia.gov/consumption/residential/data/2020/).
```

Comment:
```suggestion
Three files are included associated with RECS - a dataset with the 2015 data with some derived variables created for the book (`recs_2015`), the 2020 data with some derived variables created for the book (`recs_2020`), and the 2020 data with the original variables (`recs_2020_raw`). RECS is a survey about energy consumption and expenditure among residential households in the United States and has been conducted since 1979 by the Energy Information Administration. More information about the original data is available at the [RECS website](https://www.eia.gov/consumption/residential/data/2020/).
```